src/main/java/pl/put/poznan/transformer/texttransformers/DeleteAllCharacters.java

```diff
@@ -0,0 +1,19 @@
+package pl.put.poznan.transformer.texttransformers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Component;
+
+/**
+ * Deleting all characters
+ *
+ * @author Mikalai Chabatarevich
+ * @see TextTransformer
+ */
+@Slf4j
+@Component
+public class DeleteAllCharacters implements TextTransformer {
+    @Override
+    public String transform(String text){
+        return text.replaceAll("[^A-Za-z0-9' ']", "");
```

Comment:
Nie jestem pewien co do rozwiązania tego zadania. Według mnie trzeba by było zrobić to jako nowy endpoint gdzie użytkownik podaje nie tylko `String text` ale też tablicę znaków do usunięcia. Nie wiem czy chcemy się w to bawić, czy uznajemy że to rowiązanie jest "good enough".
@KanarXD co myślisz?